### PR TITLE
Add Reputation "Reliability" Parameter

### PR DESCRIPTION
**Pull request by DinaMeylakh**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [link to the issue](https://jira-hq.paloaltonetworks.local/browse/CIAC-6735)

## Description
Add Reputation "Reliability" Parameter to integrations with reputations that have it missing.

## Screenshots
Paste here any images that will help the reviewer

### CODE REVIEW

File: Packs/EmailRepIO/ReleaseNotes/1_0_23.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+
+#### Integrations
+
+##### EmailRep.io
+
+- Added the "Reliability" parameter for reputation commands.
```
Comment (ShahafBenYakir) [inline Packs/EmailRepIO/ReleaseNotes/1_0_23.md:6]: See the comment on the TeamCymru RN

File: Packs/TeamCymru/Integrations/TeamCymru/README.md [hunk 2]
```diff
@@ -10,7 +10,9 @@ This integration was integrated and tested with version 1.0 of TeamCymru
     | **Parameter** | **Required** |
     | --- | --- |
     | Use system proxy settings | False |
-    | Proxy URL | Supports socks4/socks5/http connect proxies \(e.g., socks5h://host:1080\). | False |    
+    | Proxy URL | Supports socks4/socks5/http connect proxies \(e.g., socks5h://host:1080\). | False |
```
Comment (ShahafBenYakir) [inline Packs/TeamCymru/Integrations/TeamCymru/README.md:13]: I think we are also missing the reputation as output in both commands

File: Packs/EmailRepIO/Integrations/EmailRepIO/EmailRepIO.py [hunk 3]
```diff
@@ -238,5 +243,5 @@ def main() -> None:
 ''' ENTRY POINT '''
 
 
-if __name__ in ('__main__', '__builtin__', 'builtins'):
+if __name__ in ('__main__', '__builtin__', 'builtins'):  # pragma: no cover
```
Comment (ShahafBenYakir) [inline Packs/EmailRepIO/Integrations/EmailRepIO/EmailRepIO.py:246]: You are failing on ut coverage?

File: Packs/DeHashed/ReleaseNotes/1_1_15.md [hunk 4]
```diff
@@ -0,0 +1,7 @@
+
+#### Integrations
+
+##### DeHashed
+
+- Added the "Reliability" parameter for reputation commands.
```
Comment (ShahafBenYakir) [inline Packs/DeHashed/ReleaseNotes/1_1_15.md:6]: See the comment on the TeamCymru RN

**Review by ShahafBenYakir — COMMENTED** [review]
Good job

File: Packs/TeamCymru/Integrations/TeamCymru/TeamCymru.yml [hunk 5]
```diff
@@ -21,6 +21,20 @@ configuration:
   type: 0
   section: Connect
   advanced: true
+- additionalinfo: Reliability of the source providing the intelligence data.
```
[the remark this answers is not in the document]
Comment (DinaMeylakh) [reply Packs/TeamCymru/Integrations/TeamCymru/TeamCymru.yml:24]: already there, check lines 82-84

File: Packs/TeamCymru/Integrations/TeamCymru/README.md (hunk 2 above)
Comment (DinaMeylakh) [reply Packs/TeamCymru/Integrations/TeamCymru/README.md:13]: already there, check lines 48, 135.

File: Packs/EmailRepIO/Integrations/EmailRepIO/EmailRepIO.py (hunk 3 above)
Comment (DinaMeylakh) [reply Packs/EmailRepIO/Integrations/EmailRepIO/EmailRepIO.py:246]: yeah..

**Review by ShahafBenYakir — APPROVED** [review]

**Pull request merged**